lib/states/il/chicago_idot.R

```diff
@@ -0,0 +1,100 @@
+source(here::here("lib", "common.R"))
+
+# Test:
+DATA_DIR <- "~/opp/data/states/il/chicago/raw_csv/"
+tmp <- load_raw(DATA_DIR, 3*10^6)
+tmp2 <- tmp %>%
+  clean()
+
+# Search logic:
+tmp$data %>%
+  select(card_no, ends_with("searched"), contraband_found_i, ends_with("found")) %>%
+  # convert Y/N/NA columns to T/F
+  mutate_at(vars("contraband_found_i", ends_with("searched")), ~if_else(is.na(.) | . == "N", F, T)) %>%
+  # convert each search result column from character to logical
+  mutate_at(vars(ends_with("found")), ~if_else(is.na(.), F, T)) %>%
+  # aggregate search, contraband_found columns
+  mutate(searched_colwise = rowSums(select(., ends_with("searched")), na.rm=T),
+         searched_colwise = searched_colwise != 0, 
+         contraband_found_colwise = rowSums(select(., ends_with("found")), na.rm=T),
+         contraband_found_colwise = contraband_found_colwise != 0) %>%
+  # Check:
+  # - Is contraband_found_i true only if searched_colwise true? => Yes
+  # - Is contraband_found_i true true if and only if contraband_found_colwise true? => Yes
+  count(searched_colwise, contraband_found_i, contraband_found_colwise)
```

Comment:
All of this testing stuff should be factored out into different file and ultimately not merged. At the moment, all of this will run every time `opp_process` is called. This file should ultimately contain just `load_raw` and `clean` functions.